developers/weaviate/model-providers/ollama/generative.md

```diff
@@ -105,7 +105,13 @@ As this integration connects to a local Ollama container, no additional credenti
 
 </Tabs>
 
-The [default model](#available-models) is used if no model is specified.
+If you don't specify a generative model, the collection uses the [default model](#available-models).
+
+import APIEndpoint from '/_includes/integrations/ollama/api-endpoint.mdx';
+
+<APIEndpoint/>
+
+For more information on configuring a generative model, see [Specify a generative model](/developers/weaviate/manage-data/collections#specify-a-generative-module).
```

Comment:
Not sure what you mean here, moving the link to the beginning? 

I think it's the same concern as above. Earlier links risk misdirecting users to the other page instead of keeping them focussed here.

Reply:
The redirect moves the user away from where they need to be. Theredirected page doesn't have the API endpoint information 

We can open up another PR to update the other pages for consistency.

Reply:
If you insist on making this change could we please do it here so we don't forget?

Once that's done pls feel free to merge. Thanks!